GoogleForCreators/web-stories-wp
# When removing a layer from a group, it can appear still inside the group

## Context

> If you remove a middle layer from a 3+ element group using the RCM option "Remove layer from group", it is still z-indexed "inside" the group.

See: https://github.com/GoogleForCreators/web-stories-wp/issues/11785



## Summary

Updates the layer ungroup code to move selected layer above it's previous group

Demo: https://www.youtube.com/watch?v=Rx3u0mteWDw



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Add 3 elements to a group
2. In the layer panel, select the middle element, right-click and select "Remove layer from group"
3. Observe that the layer selected moves to the spot above the group the item was in.


## Reviews

### Does this PR have a security-related impact?

No



### Does this PR change what data or activity we track or use?

No



### Does this PR have a legal-related impact?

No



## Checklist




---



Fixes #11785
